lib/src/FileWriter.cpp

```diff
@@ -261,13 +261,14 @@ FileWriter::calculateLogFileNameForIndex(int index)
         }
     }
 
-
-    return m_LogfileInfo.basename + patt + "." + m_LogfileInfo.extension;
+    bool separatorPresent = (m_LogfileInfo.path.endsWith(QDir::separator()) || m_LogfileInfo.basename.startsWith(QDir::separator()));
```

Comment:
Basename could be passed (unlikely) like /logfile.txt to log in root dir

Reply:
ok, but wouldn't be better to split `/logfile.txt` in `/` + `logfile.txt` ?

Reply:
i agree with you,
however since this is mostly cosmetic (double // does not affect operation on unix-like systems)
and there are many places to check at the moment i would open a new issue for this refactor.